kazoo_amqp/priv/couchdb/schemas/kapi.crm.caller_info_reply.json

```
@@ -15,6 +15,9 @@
             "enum": [
                 "caller_info_reply"
             ]
+        },
+        "Query-Interaction-Response": {
+            "type": "string"
```

It looks like the response type is object? https://github.com/2600hz/kazoo-crm/pull/95/files#diff-2bcb8628ac724d956f301c55bc810c8cb3ba33a118e44de3d22d8a9560c0c7acR128
```suggestion
            "type": "object"
```